lib/web_ui/lib/src/engine/surface/backdrop_filter.dart

```diff
@@ -65,14 +65,41 @@ class PersistedBackdropFilter extends PersistedContainerSurface
       _invertedTransform = Matrix4.inverted(_transform);
       _previousTransform = _transform;
     }
-    final ui.Rect rect = transformLTRB(_invertedTransform, 0, 0,
-        ui.window.physicalSize.width, ui.window.physicalSize.height);
+    // https://api.flutter.dev/flutter/widgets/BackdropFilter-class.html
```

Comment:
For performance reasons the `apply()` is not called when the surface is added using `addRetained`. You also need to override `retain()` to schedule this logic.

Reply:
Modified update.